core/src/main/java/org/lflang/federated/launcher/PyBuildConfig.java

```diff
@@ -12,26 +12,12 @@ public PyBuildConfig(
   }
 
   @Override
-  public String localExecuteCommand() {
-    return "python3 "
-        + fileConfig.getSrcGenPath()
-        + "/"
-        + federate.name
-        + "/"
-        + federate.name
-        + ".py -i $FEDERATION_ID";
+  public String remoteExecuteCommand() {
+    return "bin/" + fileConfig.name + "_" + federate.name + " -i '$FEDERATION_ID'";
```

Comment:
The slash here doesn't look platform independent, but the code that this replaces also featured slashes...

Reply:
Created #1917.